src/plugins/common/table/lib/table.js

```diff
@@ -114,6 +114,17 @@ define([
 		// set the table attribut "obj" as a jquery represenation of the dom-table
 		this.obj = jQuery( table );
 
+		// check config if table should be wrapped
+		if( tablePlugin.settings.wrapClass !== undefined ) {
+			// add div with class from config
+			this.obj.wrap('<div class="' + tablePlugin.settings.wrapClass + '"></div>');
```

Comment:
Check whether wrapping div already exists.
* If exists and classes match, do nothing
* If exists and classes do not match, change classes
* If exists and should not, remove
* If not exists and should, create